aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml

```diff
@@ -14,7 +14,7 @@
 
 apiVersion: v1
 name: kubernetes-dashboard
-version: 2.2.0
+version: 2.2.1
```

Comment:
Could you please bump minor version instead of patch since it is a new feature according to semver?